### PR TITLE
NN-3883: Correct page widths and general styling tidy

### PR DESCRIPTION
**Pull request by natclamp-moj**

### CODE REVIEW

File: server/views/pages/confirmedOnReport.njk [hunk 1]
```diff
@@ -18,42 +18,41 @@
 {% endblock %}
 
 {% block content %}
-  <div class="govuk-grid-row">
```
Comment (richardpopple) [inline server/views/pages/confirmedOnReport.njk:21]: Does this make the print page look odd? I'll spin it up locally and have a look

File: server/views/pages/confirmedOnReport.njk [hunk 2]
```diff
@@ -63,11 +62,13 @@
...
         }
       }) }}
-      </div>
     </div>
     <div class="print-only" data-qa="notification-of-being-on-report-section">
       {{ noticeOfBeingPlacedOnReport(noticeOfBeingPlacedOnReportData) }}
     </div>
   </div>
+  <div class="print-only">
+    {{ noticeOfBeingPlacedOnReport(noticeOfBeingPutOnReportData) }}
```
Comment (richardpopple) [inline server/views/pages/confirmedOnReport.njk:71]: This needs to be noticeOfBeingPlacedOnReportData

Comment (natclamp-moj) [reply server/views/pages/confirmedOnReport.njk:71]: Sorry it must have been the merge conflict I resolved (but wrongly)

Comment (natclamp-moj) [reply server/views/pages/confirmedOnReport.njk:71]: <div class="print-only" data-qa="notification-of-being-on-report-section">
      {{ noticeOfBeingPlacedOnReport(noticeOfBeingPlacedOnReportData) }}
    </div>
    
    is what's on main

**Pull request closed without merging**